### ISSUE TITLE
fontconfig build failed with Python 3.9 & VS2019 on Windows 10

### ISSUE THREAD

**Issue by yihuajack** [wingtk/gvsbuild#411]
Running
 command
```
python .\build.py build -p x64 gtk3 --msys-dir='E:\Program_Files\msys64' --vs-install-path='D:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise' --vs-ver=16
```
following the instructions but failed with error:
```
Traceback (most recent call last):
  File "C:\gtk-build\github\gvsbuild\gvsbuild\utils\builder.py", line 492, in build
    if self.__build_one(p):
  File "C:\gtk-build\github\gvsbuild\gvsbuild\utils\builder.py", line 618, in __build_one
    skip_deps = proj.build()
  File "C:\gtk-build\github\gvsbuild\gvsbuild\projects.py", line 347, in build
    content = f.read()
UnicodeDecodeError: 'gbk' codec can't decode byte 0xbf in position 2: illegal multibyte sequence
Error: fontconfig build failed
```
A more complete log file is attached:
[gvsbuild-log.txt](https://github.com/wingtk/gvsbuild/files/5900121/gvsbuild-log.txt)

**Comment by boondaburrah** [comment]
Probably this is related to the system encoding (as was the case when I had this issue). My build failed in exactly the same place. I made a pull request here: #408 

Try opening the file and changing the `with open()`s to force UTF-8 encoding and see if that works.

**Comment by danyeaw** [comment]
Fixed by #408.
